src/server/routes/search.js

```diff
@@ -1,10 +1,84 @@
+const algoliasearch = require('algoliasearch');
+const _ = require('lodash');
+
+const hitsPerPage = 5;
+const applicationId = '0R8E5SPAFP';
+const apiKey = '2ee58f908e6d720c4f72a9645ce0e857'; // search-only API key
```

Comment:
See previous comment about `apiKey` and `appId` and having them in the config file. Also we should be consistent with the names (`applicationId` vs `appId`).